releng/org.eclipse.cdt.target/cdt-baseline.target

```diff
@@ -51,7 +51,7 @@
       <unit id="org.eclipse.tools.templates.ui" version="0.0.0"/>
     </location>
     <location includeAllPlatforms="true" includeConfigurePhase="true" includeMode="slicer" includeSource="false" type="InstallableUnit">
-      <repository location="https://download.eclipse.org/tools/cdt/releases/cdt-lsp-3.1/cdt-lsp-3.1.0/"/>
+      <repository location="https://download.eclipse.org/tools/cdt/releases/cdt-lsp-3.2/cdt-lsp-3.2.0/"/>
```

Comment:
I can see no cdt-lsp-3.3 RC builds at present. Are we sticking with CDT-LSP 3.2.0 for 2025-09?

Reply:
Personally I do not support this cyclic reference, but they say this is to make possible a scenario where someone want to import only one CDT project to workspace and have it compiled. Well, it can work for some cases, but not for everything.

Answering your question I would align the whole target to one SimRel:
1) "latest released" for baseline target
2) "latest" for current target